augur/tasks/github/detect_move/core.py

```diff
@@ -44,6 +60,23 @@ def ping_github_for_repo_move(augur_db, key_auth, repo, logger,collection_hook='
         logger.warning(f"Could not check if repo moved because the api timed out 10 times. Url: {url}")
         return
 
+    
+    
+    #Mark as errored if not found
+    if response_from_gh.status_code == 404:
+        logger.error(f"Repo {repo.repo_git} responded 404 when pinged!")
+
+        repo_update_dict = {
+        'repo_git': repo.repo_git,
+        'repo_path': None,
+        'repo_name': None,
+        'description': f"During our check for this repo on {datetime.today().strftime('%Y-%m-%d')}, a 404 error was returned. The repository does not appear to have moved. Instead, it appears to be deleted"
```

Comment:
What is the current equivalent of setting the status to 'New' for recloning? We should probably do that as well. Maybe its in here and we don't see it?

Reply:
detect_move is only used for non-facade tasks. Facade has it's own logic to handle if repos can't be cloned. Facade set's the status to 'Failed Clone' if we can't clone it.

Reply:
What if its deleted after we have already cloned it and populated commits? Should we have a different status for that?

Reply:
I don't think it makes much of a difference. We could make an extra status, something like 'Stale', to be more clear about the repo's status in the database.

Reply:
I think the thing that's important is making sure we don't get a singular network issue that marks all the repositories some kind of "stale'.